web/src/test/java/org/cbioportal/genome_nexus/web/CancerHotspotsIntegrationTest.java

```diff
@@ -22,18 +27,40 @@
     webEnvironment = SpringBootTest.WebEnvironment.DEFINED_PORT,
     properties = {
         "vep.url=http://grch37.rest.ensembl.org/vep/human/hgvs/VARIANT?content-type=application/json&xref_refseq=1&ccds=1&canonical=1&domains=1&hgvs=1&numbers=1&protein=1",
-        "hotspots.url=https://www.cancerhotspots.org/api/hotspots/single/",
-        "hotspots.3d.url=https://www.3dhotspots.org/api/hotspots/3d/",
         "spring.data.mongodb.uri=mongodb://localhost/integration",
         "server.port=38888"
     }
 )
 public class CancerHotspotsIntegrationTest
 {
+    private static List<Hotspot> HOTSPOTS;
+
+    @BeforeClass
+    public static void readMockFile() throws IOException
+    {
+        JsonToObjectMapper objectMapper = new JsonToObjectMapper();
+        // reading the entire DB dump for a full scale test
+        HOTSPOTS = objectMapper.readHotspots("hotspots_db_dump.json");
```

Comment:
Reading a `json` dump into memory first as `Hotspot` instances, and then inserting into the DB. There might be a faster, more straightforward way to do this.

Reply:
Yeah also we would have to keep the dump in sync. It's most comprehensible I think to only import the relevant data for testing, so it's easy to see what kind of data is expected. You can use e.g. `jq` to get the relevant data.

Or if necessary to have the full dump, do a complete integration and use the actual data that we would import instead of having to sync this file.

Reply:
@inodb At this point I don't think we need the full dump. For now let's include only the hotspots related to the test then.